lib/assessment.rb

```diff
@@ -21,14 +32,24 @@
 
 ##
 # your code here
+class Person
+  attr_accessor :name :location
+  def initialize(name,age,location)
+      @name = name
+      @age = age
+      @location = location
+    end
+  end
 ##
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = dave.new 32 "Ohio" # replace nil with your code
+Answer.dave.location =
```

Comment:
You needed to `Answer.dave.location = "location here"` where you use the correct location.